.pylintrc

```diff
@@ -209,7 +209,10 @@ disable=raw-checker-failed,
         single-string-used-for-slots,
         fixme,
         import-error,
-        unspecified-encoding
+        unspecified-encoding,
+        use-dict-literal,
```

Comment:
After pylint upgrade, new messages started to show up. The more interesting one is **use-dict-literal**. Basically, it is recommended to use a literal dictionary using curly brackets {...}, rather than dict() as it is quicker (similarly to the f-string vs string case)